frontend/src/app/main/features.cljs

```diff
@@ -90,46 +76,28 @@
         state
         (do
           (log/trc :hint "feature enabled" :feature feature)
-          (update state :features-runtime (fnil conj #{}) feature))))))
+          (-> state
+              (update :features-runtime (fnil conj #{}) feature)
+              (update :features-runtime set/intersection cfeat/no-migration-features)))))))
 
 (defn initialize
-  ([] (initialize #{}))
-  ([team-features]
-   (assert (set? team-features) "expected a set of features")
-   (assert (every? string? team-features) "expected a set of strings")
-
-   (ptk/reify ::initialize
-     ptk/UpdateEvent
-     (update [_ state]
-       (let [runtime-features (get state :features/runtime #{})
-             team-features    (into #{}
-                                    cfeat/xf-supported-features
-                                    team-features)]
-         (-> state
-             (assoc :features-runtime runtime-features)
-             (assoc :features-team team-features))))
-
-     ptk/WatchEvent
-     (watch [_ _ _]
-       (when *assert*
-         (->> (rx/from cfeat/no-migration-features)
-              ;; text editor v2 isn't enabled by default even in devenv
-              ;; wasm render v1 isn't enabled by default even in devenv
-              (rx/filter #(not (or (contains? cfeat/backend-only-features %)
-                                   (= "text-editor/v2" %)
-                                   (= "render-wasm/v1" %)
-                                   (= "design-tokens/v1" %))))
-              (rx/observe-on :async)
-              (rx/map enable-feature))))
```

Comment:
With this removed, we will no longer have `no-migration-features` automatically enabled in dev environment?

Reply:
Yes, thats it. It is continue source of confusion that why some feature is autoenabled. It is better just to remove this and if you work with a feature, just enable it. Just look, all ongoing features were excluded from activation

Reply:
ok. Only tell people about the change.